packages/react-server/package.json

```diff
@@ -33,14 +33,18 @@
     "@shopify/react-hydrate": "^1.1.4",
     "@shopify/react-network": "^3.3.1",
     "@shopify/sewing-kit-koa": "^6.1.2",
+    "@shopify/useful-types": "^1.3.0",
     "chalk": "^2.4.2",
     "koa": "^2.5.0",
     "koa-compose": "^4.1.0",
-    "koa-mount": "^4.0.0"
+    "koa-mount": "^4.0.0",
+    "fs-extra": "^8.1.0",
+    "isomorphic-fetch": "2.2.1"
```

Comment:
Is this actually used for it directly?

Reply:
It isn't no.

Reply:
Might make more sense as a peer dependency.